api/nodecontrol/auction-request.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "io.nats.nex.v2.auction_request",
+  "title": "AuctionRequest",
+  "type": "object",
+  "properties": {
+    "arch": {
+      "type": "string",
+      "description": "The architecture auction for",
```

Comment:
this description seems awkward with `for` at the end. Maybe `Specify a required architecture for the auction`